### PR TITLE
Fix SAML login to just go to one URL fixes #129

### LINKED ISSUE #129: Fix SAML login to just go to one URL
All we need to do is add a config parameter to the URL

### PR DESCRIPTION
**Pull request by jonespm**

### CODE REVIEW

**Comment by jonespm** [comment]
This adds a django settings module, which will also allow us to easily fix #92

File: dashboard/settings.py [hunk 1]
```diff
@@ -285,3 +291,5 @@
     AUTHENTICATION_BACKENDS += ('django.contrib.auth.backends.ModelBackend',)
     LOGIN_REDIRECT_URL = '/'
     
+
+SETTINGS_EXPORT = ['LOGIN_URL','LOGOUT_URL','DEBUG']
```
Comment (zqian) [inline dashboard/settings.py]: What's the purpose of this setting?

Comment (zqian) [reply dashboard/settings.py]: I see, it is used in base.html.

Comment (jonespm) [reply dashboard/settings.py]: Yeah I can add a comment here, it's for the settings module. Basically every setting we want to have available to in the templates needs to be in here. I added DEBUG as I figured we'd want that.

**Pull request merged**